src/api/test/functional/webui/package_controller_test.rb

```diff
@@ -329,6 +329,20 @@ def test_submit_package
 
     # modify and resubmit
     Suse::Backend.put( '/source/home:adrian/x11vnc/DUMMY?user=adrian', 'DUMMY')
+
+    click_link 'Submit package'
```

Comment:
What about `spec/controllers/webui/package_controller_spec.rb` and `spec/features/webui/requests_spec.rb`?